meta-mender-core/classes/mender-part-images.bbclass

```diff
@@ -43,6 +43,8 @@ IMAGE_NAME_SUFFIX = ""
 # Block storage
 ################################################################################
 
+MENDER_RFS_PART_EMPTY_B ??= "n"
```

Comment:
We should follow the existing naming, IOW this should be `MENDER_ROOTFS_PART_B_EMPTY`.